src/wallOverlap.cpp

```diff
@@ -62,6 +62,7 @@ float WallOverlapComputation::getFlow(const Point& from, const Point& to)
         //           to other
 
         bool are_in_same_general_direction = dot(from - to, to_other_it.p() - to_other_next_it.p()) > 0;
+#if 0
```

Comment:
Please don't disable code this way. If it's no longer needed, remove it.

Reply:
Hi, that's understood. But as the title says, this PR is up for discussion. I don't want to completely remove it yet because I don't feel 100% confident that I actually understand why it is there in the first place. I would very much like for people who understand how the original code worked to consider the changes I have made and suggest how this could move forward. I'm a bit stuck now.

Reply:
We've got our Git history for that if necessary :wink: Feel free to remove it.

That said, we will take a better look at it. Nallath's review was just something he noticed when passing by the notification in his e-mail, I think.